### ISSUE TITLE
InversionTime

### ISSUE THREAD

**Issue by neurolabusc** [bids-standard/bep001#86]
The specification suggests InversionTime should be sourced from DICOM tag [0018,0082](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0082)). However, some modern enhanced DICOMs store inversion time in tag [0018,9079](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,9079)). The latter allows an array of multiple values, but some vendors (e.g. Siemens XA) use this for single values as well. I would update the document to list either source.

**Comment by agahkarakuzu** [comment]
@neurolabusc if there are acquisitions varying with respect to `TR`, the convention is to put each value in separate jsons per image. In this case, `Inversion Time` looks OK as is. 

But if `Inversion Time` and `Inversion Times` are mutually exclusive, then we may note that.

**Comment by neurolabusc** [comment]
Agreed. Prior versions of dcm2niix followed BIDS and only populated `InversionTime` if  0018,0082 exists, the upcoming release will populate it if either 0018,0082 exists or if 0018,9079 exists with a single value. This should help users of Siemens XA systems. I do not have access to Siemens XA, so I am not sure if it ever creates an enhanced DICOM where more than one TI is listed for a single file. Since I have no exemplars, dcm2niix will generate a warning if this ever occurs. 

My only comment here is that I think the bmp should be updated to list multiple possible sources for this useful parameter.

**Comment by agahkarakuzu** [comment]
Thanks @neurolabusc, I agree! I will make a note for this case.
